gbfs.md

```diff
@@ -707,36 +706,36 @@ All stations included in `station_information.json` are considered public (meani
+`stations[].rental_uris.ios` | OPTIONAL | URI | URI that can be used on iOS to launch the rental app for this station. More information on this iOS feature can be found [here](https://developer.apple.com/documentation/uikit/core_app/allowing_apps_and_websites_to_link_to_your_content/communicating_with_other_apps_using_custom_urls?language=objc). Please use iOS Universal Links (https://developer.apple.com/ios/universal-links/) if possible so viewing apps do not need to manually manage the redirect of the user to the app store if the user does not have the application installed. <br><br>This URI SHOULD be a deep link specific to this station, and SHOULD NOT be a general rental page that includes information for more than one station.  The deep link SHOULD take users directly to this station, without any prompts, interstitial pages, or logins. Make sure that users can see this station even if they never previously opened the application.  <br><br>If this field is empty, it means deep linking is not supported in the native iOS rental app. <br><br>Note that the URI does not necessarily include the `station_id` for this station - other identifiers can be used by the rental app within the URI to uniquely identify this station. <br><br>See the [Analytics](#analytics) section for how viewing apps can report the origin of the deep link to rental apps. <br><br>iOS Universal Links example value: `https://www.example.com/app?sid=1234567890&platform=ios` <br><br>Deep Link (without Universal Links) example value: `com.example.ios://open.example.app/app?sid=1234567890`
+`stations[].rental_uris.web` | OPTIONAL | URL | URL that can be used by a web browser to show more information about renting a vehicle at this station. <br><br>This URL SHOULD be a deep link specific to this station, and SHOULD NOT be a general rental page that includes information for more than one station.  The deep link SHOULD take users directly to this station, without any prompts, interstitial pages, or logins. Make sure that users can see this station even if they never previously opened the application.  <br><br>If this field is empty, it means deep linking is not supported for web browsers. <br><br>Example value: `https://www.example.com/app?sid=1234567890`
```

Comment:
here as well

Reply:
rental_uris is not an array, so no need for [ ].

Reply:
The sentence `Please use iOS Universal Links if possible so viewing apps do not need to manually manage the redirect of the user to the app store if the user does not have the application installed` is not correct. Universal Links do not fallback to the App Store if the app is not installed. Instead they open the website in the browser. That website in turn could redirect to the App Store but this is not a feature of Universal Links per se. So I would write: "Please use iOS Universal Links if possible so viewing apps provide a fallback web experience if the application is not installed."

Reply:
@futuretap I think you should open an issue about iOS links

Reply:
@tdelmas done: https://github.com/MobilityData/gbfs/pull/585